<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Piano</class>
 <widget class="QWidget" name="Piano">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1490</width>
    <height>907</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>1441</width>
     <height>821</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1439</width>
      <height>819</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
  </widget>
  <widget class="QPushButton" name="del_all">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>удалить всё</string>
   </property>
  </widget>
  <widget class="QPushButton" name="play_melody">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>50</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>воспроизвести мелодию</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cells">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>50</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>добавить</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="count_cells">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>50</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>42</width>
     <height>22</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="YES">
   <property name="geometry">
    <rect>
     <x>1250</x>
     <y>50</y>
     <width>231</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>соxранить мелодию(формат wav)</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="error">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>20</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
